lib/reline/line_editor.rb

```diff
@@ -803,7 +803,7 @@ def add_dialog_proc(name, p, context = nil)
       end
       contents = contents[dialog.scroll_top, height]
     end
-    if dialog_render_info.scrollbar and dialog_render_info.contents.size > height
+    if dialog_render_info[:scrollbar] and dialog_render_info.contents.size > height
```

Comment:
I think this is a mis-replacement caused by replacing `face.scrollbar` to `face[:scrollbar]`

Reply:
Thanks https://github.com/ruby/reline/pull/552/commits/e7a6d562788e3f4088c1682add60270337464d03